packages/parser/src/ASTParser.ts

```diff
@@ -184,6 +184,15 @@ class ASTParser {
               ASTParser.logEntry(`[Info] Open jsxElement`);
               jsxElement.open(node);
             } else {
+              if (node.name.type === 'JSXIdentifier') {
+                if (
+                  jsxElement.isRoute() &&
+                  attribute.getElementName() == 'element'
+                ) {
+                  jsxElement.isRouteElement = true;
+                  jsxElement.setName(node.name.name);
+                }
+              }
               if (
                 jsxElement.isRoute() &&
                 attribute.getElementName() == 'render'
```

Comment:
Instead of adding line 187-195, you could consider modifying the existing if statement to:

```
if (jsxElement.isRoute() && (attribute.getElementName() === 'render' || attribute.getElementName() === 'element')
```